src/nfts/NftMedia.tsx

```diff
@@ -58,64 +67,98 @@ export default function NftImage({
   origin,
   ErrorComponent,
   testID,
+  mediaType,
 }: Props) {
   const [status, setStatus] = useState<Status>(!nft.metadata ? 'error' : 'loading')
-  const [scaledHeight, setScaledHeight] = useState(DEFAULT_IMAGE_HEIGHT)
+  const [scaledHeight, setScaledHeight] = useState(DEFAULT_HEIGHT)
   const [reloadAttempt, setReloadAttempt] = useState(0)
+  const headerHeight = useHeaderHeight()
 
   const fetchingNfts = useSelector(nftsLoadingSelector)
 
   const imageUrl = nft.media.find((media) => media.raw === nft.metadata?.image)?.gateway
+  const videoUrl = nft.media.find((media) => media.raw === nft.metadata?.animation_url)?.gateway
 
   useEffect(() => {
     if (nft.metadata) {
       setStatus('loading')
     } else {
-      sendImageLoadEvent('No nft metadata')
+      sendLoadEvent('No nft metadata')
       setStatus('error')
     }
   }, [`${nft.contractAddress}-${nft.tokenId}`])
 
+  // if the image failed to load before, try again when the user pulls to refresh
   useEffect(() => {
-    // if the image failed to load before, try again when the user pulls to refresh
     if (status === 'error' && fetchingNfts) {
       setStatus('loading')
       setReloadAttempt((prev) => prev + 1)
     }
   }, [status, fetchingNfts])
 
-  function sendImageLoadEvent(error?: string) {
+  function sendLoadEvent(error?: string) {
```

Comment:
nit: Maybe it's cleaner to split this function into a `sendSuccessfulEvent` and a `sendErrorEvent`.